canmatrix/sym.py

```diff
@@ -162,7 +162,10 @@ def sendreceive(f):
         for frame in frames:
             name = "[" + frame.name + "]\n"
 
-            idType = "ID=%08Xh" % (frame.id)
+            if frame.extended == 1:
```

Comment:
```python
if frame.extended:
```

Reply:
I think it was this issue:


https://github.com/ebroecker/canmatrix/commit/cfec9b81ecbbc7ae0ab3ab1c2c1b756abffe01d5#diff-11bf3294736079e6a427e649315ff8f6

Reply:
Oh No

I Just cannot see this issues and Changes on my Phone ...
So forget my last comment